packages/dx-react-grid/docs/reference/grouping-panel.md

```diff
@@ -43,6 +43,7 @@ Field | Type | Description
 ------|------|------------
 column | [Column](#column) | Specifies the column associated with the item.
 draft | boolean | Specifies whether the item should be rendered for the preview.
+mode | string | Specifies grouping mode "add", "remove" or "reorder".
```

Comment:
It feels a bit strange that mode is linked to the draft state but has such generic name. It also seems that draft and mode should be combined into the one property.